inferno-framework/inferno-core
# Fi 3182 inferno execute short ids

# Summary

Create command line option that allows for unified short ids `--short-ids` (alias: `-r`)

# Testing Guidance
1. checkout this branch
2. `bundle exec inferno services start`
3. verify old behavior
```
bundle exec inferno execute --suite infra_test --groups 1 --tests 4.01 --inputs suite_input:a outer_group_input:b inner_group_input:c test_input:d external_test1_input:asdf external_inner_group_input:asdf external_outer_group_input:asdf
```
4. verify new behavior
```
bundle exec inferno execute --suite infra_test --short-ids 1 4.01 --inputs suite_input:a outer_group_input:b inner_group_input:c test_input:d external_test1_input:asdf external_inner_group_input:asdf external_outer_group_input:asdf
```
5. this should raise an exception to prevent running 1.01 twice
```
bundle exec inferno execute --suite dev_validator -r 1 1.01 -i "url:https://hapi.fhir.org/baseR4" patient_id:1234321
```
